RELEASE.md

```diff
@@ -171,9 +171,30 @@ Then, the Release Manager releases and announces the new version:
 
 -   [ ] 1.  If all is good give a public OK to the release
 
--   [ ] 2.  Tag the CLI, Client Side Tools, and Coverage Reporter with the version of the release being done.
+-   [ ] 2.  Tag the CLI and Coverage Reporter with the version of the release being done.
 
--   [ ] 3.  Inform all stakeholders the release is finished 
+    **Note:** This process will be improved in [REL-51](https://codacy.atlassian.net/browse/REL-51)
+
+        1.  Go to the repository (or clone it)
```

Comment:
Codacy found an issue: [[code-block-style] Code blocks should be fenced](https://app.codacy.com/gh/codacy/chart/pullRequest?prid=6190172)